Packs/Accessdata/Integrations/AccessdataV2/AccessdataV2.yml

```diff
@@ -0,0 +1,259 @@
+commonfields:
+  id: AccessData
+  version: -1
+
+name: AccessData
+display: AccessData
+category: Forensics & Malware Analysis
+description: Use the AccessData package to integrate with the AccessData platform enabling the automation of case/evidence management and endpoint collection.
+
+configuration:
+- display: Web Protocol
+  name: PROTOCOL
+  defaultvalue: http
+  type: 0
+  required: true
+- display: Service URL (FQDN or IP Address.)
+  name: SERVER
+  defaultvalue: localhost
+  type: 0
+  required: true
+- display: Service Listening Port
+  name: PORT
+  defaultvalue: 4443
+  type: 0
+  required: true
+- display: "The API authentication key. http://support.accessdata.com/hc/en-us/articles/360053994573-Generating-API-Keys"
+  name: APIKEY
+  defaultvalue: 00000000-abcd-1234-5678-000000000000
```

Comment:
I would normally agree but the value is fake and never works. This is in place so that users understand the form of key they need to supply. If you request it to be removed, I can do so as well.

Reply:
@ThomasVieth - If you want to supply a text to hint the expected format of the value of this field, you can use `additionalinfo` key, e.g:

```yml
...
- display: "The API authentication key. http://support.accessdata.com/hc/en-us/articles/360053994573-Generating-API-Keys"
  name: APIKEY
  additionalinfo: The API authentication key has the following format `00000000-abcd-1234-5678-000000000000`
...
```

Reply:
Thanks! this is exactly what I was aiming for 😄